plugins/wikidata/__init__.py

```
@@ -226,14 +226,17 @@ def parse_wikidata_response(self, item, item_id, genre_source_type, response, re
                             for tmp in genre_entries:
                                 if tmp == node1.attribs.get('about'):
                                     list1 = node1.children.get('name')
-                                    for node2 in list1:
```

How about:
```
for node in (list1 if list1 else []):
```

---

I have no idea. I think logging the issue happened is a good idea, as if it fail silently you won't have the data and won't know why it is missing.

---

Yes - it probably is.